data/json/items/comestibles/junkfood.json

```diff
@@ -1288,5 +1330,152 @@
     "flags": [ "EATEN_HOT" ],
     "vitamins": [ [ "vitA", 3 ], [ "iron", 10 ] ],
     "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_base",
+    "name": "gelatin dessert powder",
+    "weight": "175 g",
+    "color": "white",
+    "spoils_in": "1461 days",
+    "container": "box_small",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -8,
+    "calories": 20,
+    "description": "A small box of gelatin dessert powder. Just add water and set. Comes in a variety of flavors.",
+    "price": 5,
+    "price_postapoc": 10,
+    "material": [ "flesh", "junk" ],
+    "volume": "250ml",
+    "flags": [ "EATEN_COLD" ],
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_cherry",
+    "name": "cherry gelatin dessert powder",
+    "color": "red",
+    "description": "A small box of cherry gelatin dessert powder. Just add water and set.",
+    "copy-from": "gelatin_dessert_powder_base"
+  },
```

Comment:
This and the other flavors should be compressed into snippets for a single gelatin dessert powder

Reply:
I'd be happy to change that. I'm not entirely sure how snippets work though.

Reply:
Does snippets work with recipes?

Reply:
For a reference, check out cooked mutant meat. Basically you have a generic decription in the item and an added `"snippet_category"` field, and you define a set of snippets with an id separately. A random snippet will be shown after the generic description and if you set the id up right it will persist between viewing.

Reply:
`data/json/items/comestibles/carnivore.json` has some examples of comestibles with snippets. For example, `"mutant_meat"` has snippet category `"mutant_meat_desc"`. But I don't think that will add the correct snippet when crafting. It also does not change the item's name.

Conditional names can be used to change the item's name based on its components, but that currently does not change the description.

Reply:
Thank you. I'll have a look through those sections and see what I can do to clean up and compress the different flavours after work today.